internal/customflag/multistring_test.go

```diff
@@ -29,9 +29,11 @@ func TestMultiString(t *testing.T) {
 	var bar []string
 	fs.Var(customflag.NewMultiStringValue(&bar), "bar", "Test flag #2.")
 
-	fs.Parse([]string{
+	if err := fs.Parse([]string{
```

Comment:
I think the code becomes more readable if we create an intermediate variable:

```go
args := []string{"--foo=a", "--bar=b", ...}
if err := fs.Parse(args); err != nil {
    t.Fatal(err)
}
```

See also: https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
Done.